docs/state.md

```diff
@@ -16,7 +16,7 @@ import { Text, View } from 'react-native';
 const Blink = (props) => {
   const [isShowingText, setIsShowingText] = useState(true);
 
-   useEffect(() => {
+  useEffect(() => {
      const toggle = setInterval(() => {
        setIsShowingText(!isShowingText);
      }, 1000);
```

Comment:
Good spot, removing slightly more block indentation should fully do the trick:

```diff
-   useEffect(() => {
-     const toggle = setInterval(() => {
-       setIsShowingText(!isShowingText);
-     }, 1000);
-
-     return () => clearInterval(toggle);
-  })
+  useEffect(() => {
+    const toggle = setInterval(() => {
+      setIsShowingText(!isShowingText)
+    }, 1000)
+
+    return () => clearInterval(toggle)
+  })
```

Reply:
@leotm - Thanks for the reply. I did sign the agreement...

So, for the improved formatting (I didn't notice the entire block was offset...) how should I proceed?

Thanks in advance.

Reply:
> So, for the improved formatting (I didn't notice the entire block was offset...) how should I proceed?

The best way will be to checkout repository locally and follow the contributing guide:
* https://github.com/facebook/react-native-website#-contributing

However you can also edit the file on your PR branch via GitHub UI or Codespaces.